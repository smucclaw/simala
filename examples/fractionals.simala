fractional = 0.5 * 0.5;

#eval fractional;
#eval floor(fractional);
#eval ceiling(fractional);
#eval fractional * fromInt(2)
