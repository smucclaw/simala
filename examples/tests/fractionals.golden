0.25
0
1
0.5
-0.3
9.0e-2
